src/pages/access_review/AppSummary.js

```diff
@@ -76,10 +76,6 @@ export const AppSummary = hh(class AppSummary extends React.Component {
         h(DownloadLink,{
           label: 'DUL machine-readable format',
           onDownload: () => download('machine-readable-DUL.json', mrDUL)
-        }),
-        h(DownloadLink,{
-          label: 'Data Use Letter',
-          onDownload: this.downloadDUL
```

Comment:
This function is defined on line 59 and is no longer in use - we should delete it as part of this PR. Interestingly, that function calls `Files.getDulFile(consentId, dulName);` which doesn't exist anymore, likely removed in a previous cleanup. Once you remove that, then `import { Files } from '../../libs/ajax';` also becomes unused and we can remove that too.